spec/models/track_spec.rb

```diff
@@ -246,7 +354,8 @@
       self_organized_track.cfp_active = true
       self_organized_track.save!
       @a_track_organizer.add_role 'track_organizer', self_organized_track
-      @an_event_of_the_track = create(:event, program: self_organized_track.program, track: self_organized_track)
+      @an_event_of_the_track = create(:event, program: self_organized_track.program, track: self_organized_track, state: 'confirmed')
```

Comment:
How about we change this variable name to something shorter/simpler and self-explanatory?
This is an event that has as track the self_organized_track, so some ideas:
```
event_with_self_organized_track
self_organized_track_event
event_of_self_organized_track
```